src/cascadia/TerminalApp/SuggestionsControl.xaml

```diff
@@ -206,6 +206,17 @@
                       ItemsSource="{x:Bind FilteredActions}"
                       SelectionChanged="_listItemSelectionChanged"
                       SelectionMode="Single" />
+            <mux:TeachingTip x:Name="DescriptionTip"
+                             Title=""
+                             IsOpen="False"
+                             PreferredPlacement="Right"
+                             ShouldConstrainToRootBounds="False"
+                             Subtitle="">

```

Comment:
Is "Subtitle" necessary? Why?

Reply:
Same with Title?

Also, looks like the WinUI docs don't say the default value (grr). So, mind checking each of these to make sure they're necessary?